deploy/sample-backup-configs/sample_config_build_recipes.ini

```diff
@@ -10,209 +10,55 @@
 # non-default instancetype.
 
 # Single-core, Rocket-based recipes
```

Comment:
for pedagogical purposes, can we put the vanilla configs at the top of this file, with more complex ones towards the end?

Reply:
The current ordering is 

1. Single-core Rocket+Gemmini
2. Quad-core Rocket
3. Single-core BOOM
4. Supernode
5. midasexample
6. RAM Opts

What do you prefer?

Reply:
Quad rocket
Single BOOM
Single Rocket + Gemm
RAM Opts
Supernode
Midasexample